File: src/db_to_csv.rb
require 'sequel'
require "csv"

# Run time argv
DB = Sequel.connect("sqlite://fb.sqlite")

# Load to_csv method via extension
DB.extension(:sequel_3_dataset_methods)

CSV.open("output.csv", "w") do |csv|
  csv << ["person_hash_id", "person_id", "person_name"]

  DB['SELECT * FROM People;'].each do |person|
    csv << [person[:person_hash_id], person[:person_id], person[:person_name]]
  end
end

File: src/db_to_csv.py
import simplejson
import facebook
import hashlib
import sys
import sqlite3 as lite
import contextlib
from datetime import datetime
import pdb

conn = lite.connect('./fb.sqlite')
# conn.text_factory = str
cur = conn.cursor()
data = cur.execute("SELECT * FROM People")

f = open('output.csv', 'w')
print >> f, "person_hash_id, person_id, person_name"
for row in data:
  print >> f, row
f.close()